app/features/quiz/QuizSession/logic.js

```diff
@@ -33,9 +33,8 @@ export const queueLoadLogic = createLogic({
       const limit = !queueCount
         ? INITIAL_QUEUE_LIMIT
         : wrapUp.active
-          ? wrapUp.count - queueCount
+          ? wrapUp.count
```

Comment:
This change ensures that the queue load happening when in wrapUp loads the queue up to the full wrapUp count, not just the difference (because the query limit equates to the amount loaded into the queue)